sdk/test-utils/recorder/test/testProxyTests.spec.ts

```diff
@@ -3,11 +3,12 @@
 
 import { ServiceClient } from "@azure/core-client";
 import { createPipelineRequest } from "@azure/core-rest-pipeline";
-import assert from "assert";
+import { assert } from "vitest";
 import { expect } from "chai";
```

Comment:
Why is `expect` coming from `chai` and not `vitest`?

Reply:
updated